src/main/java/io/github/jtama/openrewrite/ProjectAerialViewGenerator.java

```diff
@@ -39,23 +39,36 @@
 import io.github.jtama.openrewrite.model.LinksReport;
 import io.github.jtama.openrewrite.model.Node;
 import io.github.jtama.openrewrite.model.NodesReport;
- 
+
 /**
  * An OpenRewrite recipe that scans a Java project and generates it's internal dependency graph
  */
 public class ProjectAerialViewGenerator extends ScanningRecipe<ProjectAerialViewGenerator.@NotNull GraphScanAccumulator> {
 
     @Option(displayName = "Maximum nodes", description = "The maximum number of nodes to display in the graph. We will try to retain the largest nodes.", required = false)
```

Comment:
I think it's more readable o let the annotations at the field level, and add getters, default constructor and fulle constructor annotated with the constructor `@com.fasterxml.jackson.annotation.JsonCreator`